benchmarks/templates/benchmarks/tutorials/models/quickstart.html

```diff
@@ -38,6 +38,17 @@ <h3 class="benefits_heading is-size-3-mobile">Let's Jump Right In</h3>
     <div class="columns is-tablet is-variable is-1-tablet">
          <div class="column is-one-half">
             <pre><code>
+# Install and activate Miniconda
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh -b
+~/miniconda3/bin/conda init
+source ~/.bashrc
```

Comment:
This is Linux specific. I think a[ link to Conda ](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)might be more useful and universal.

For Mac, wget would need to be replaced by `curl` and everything else would theoretically be the same.
For Windows, a `.exe` installer would need to be used first.

Reply:
I made this update.  Should just take a minute for your review.